src/main/java/com/artipie/asto/key/KeyParts.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.key;
+
+import com.artipie.asto.Key;
+import org.cactoos.list.ListEnvelope;
+import org.cactoos.list.ListOf;
+
+/**
+ * Parts of a key.
+ *
+ * @since 1.8.1
+ */
+public final class KeyParts extends ListEnvelope<String> {
```

Comment:
I'm not sure that we need a separate class for this. Maybe we can add a method to `Key` (because each key implementation could be split in parts, and the delimiter is a part of contract). @artipie/members WDYT?

Reply:
@g4s8 I was thinking about that ...

Reply:
@artipie/members I think it will be nice and useful to add to `Key` interface this :
```java
public interface Key {

    /**
     * Delimiter used to split string into parts and join parts into string.
     */
    String DELIMITER = "/";
    
    /**
     * Parts of key.
     * @return List of parts
     */
    List<String> parts();
```

@olenagerasimova WDYT ?